modules/tensor_mechanics/src/userobjects/GeneralizedPlaneStrainUserObject.C

```diff
@@ -38,7 +38,9 @@ GeneralizedPlaneStrainUserObject::GeneralizedPlaneStrainUserObject(
     _Cijkl(getMaterialProperty<RankFourTensor>(_base_name + "elasticity_tensor")),
     _stress(getMaterialProperty<RankTwoTensor>(_base_name + "stress")),
     _out_of_plane_pressure(getFunction("out_of_plane_pressure")),
-    _factor(getParam<Real>("factor"))
+    _factor(getParam<Real>("factor")),
+    _residual(1),
+    _jacobian(1)
```

Comment:
It isn't actually known at compile time, so I think a vector is actually the right thing to use here.  It will always be size 1 in this base class, but we're going to make a class that derives from it in BISON that will have a variable size.